.env.prod

```diff
@@ -21,3 +21,4 @@ EOS_PORT_DEFAULT='443'
 EOS_CHAINID_DEFAULT='4667b205c6838ef70ff7988f6e8257e8be0e1284a2f59699054a018f743b1d11'
 NEW_ENDPOINT_PERIOD=600000
 APP_LOCATION='https://peeranha.io'
+SINGLE_COMMUNITIES="2,https://ontology.peeranha.io"
```

Comment:
Single line configuration is hard to manage as we add more communities and add more values to the config. Can we configure name of configuration file here instead?

For example:

`COMMUNITIES_CONFIG_FILE_NAME='communities-config-prod'`

and the in the import:

`import communitiesConfig from process.env.COMMUNITIES_CONFIG_FILE_NAME;`

Reply:
@sergeyi23 I refacroted cookie utils.
About communities config. I fully agreed with you about one line config. Also it's impossible to use variable as path. So created mapped object at community config file which contains env values as fields and communities as values